src/main/java/scrumweb/user/project/controller/ProjectController.java

```diff
@@ -0,0 +1,24 @@
+package scrumweb.user.project.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import scrumweb.dto.ProjectDto;
+import scrumweb.user.project.service.ProjectService;
+
+import static scrumweb.common.ApplicationConstants.API_URL;
+
+@RestController
+@RequestMapping(API_URL + "profile")
+public class ProjectController {
+
+    @Autowired
+    protected ProjectService projectService;
+
+    @PostMapping("/createproject")
```

Comment:
zmień na /create, bo jest project kontroller i wiadomo, że tworzymy projekt :)

Reply:
tylko, że url będzie /api/scrum-web/profile/create a to raczej sugeruje tworzenie profilu niż tworzenie projektu ewentualnie można zrobić /api/scrum-web/profile/projects/create